src/lamp_py/ingestion/ingest.py

```diff
@@ -76,9 +64,8 @@ def ingest_files(
 
     # GTFS Static Schedule must be processed first for performance manager to
     # work as expected
-    if ConfigType.SCHEDULE in converters:
-        converters[ConfigType.SCHEDULE].convert()
-        del converters[ConfigType.SCHEDULE]
+    gtfs_converter = GtfsConverter(ConfigType.SCHEDULE, metadata_queue)
+    gtfs_converter.convert()
```

Comment:
i wouldn't be opposed to putting this into `pipeline.py`. i kind of like the idea of the main method having calls to multiple ingestion methods for gtfs_rt, schedule, and glides.

Reply:
I think it's ok to leave here for now, since the ingestion of these types is linked in sequence.